projects/plugins/jetpack/_inc/client/pro-status/index.jsx

```diff
@@ -190,20 +191,19 @@ class ProStatus extends React.Component {
 			pluginSlug = 'akismet/akismet.php';
 		}
 
-		const hasPersonal = /jetpack_personal*/.test( sitePlan.product_slug ),
-			hasFree = /jetpack_free*/.test( sitePlan.product_slug ),
-			hasBackups = get( vpData, [ 'data', 'features', 'backups' ], false ),
-			hasVPScan = get( vpData, [ 'data', 'features', 'security' ], false );
-
-		const { scanStatus } = this.props;
+		const hasFree = /jetpack_free*/.test( sitePlan.product_slug ),
```

Comment:
Is there any reason why this cannot be replaced with a feature check?

Reply:
> We were going for smaller steps

That's fine, only wanted to double check whether we had any other restriction 🙂.